test/gurka/test_time_tracking.cc

```diff
@@ -254,6 +255,27 @@ TEST(TimeTracking, routes) {
   }
 
   ASSERT_THAT(times, testing::Pointwise(testing::DoubleNear(0.0001), expected));
+
+  // matrix between them
+  req =
+      boost::format(
+          R"({"costing":"auto","sources":[{"lon":%1%,"lat":%2%},{"lon":%3%,"lat":%4%},{"lon":%5%,"lat":%6%}],"targets":[{"lon":%7%,"lat":%8%},{"lon":%9%,"lat":%10%}]})") %
+      map.nodes["A"].first % map.nodes["A"].second % map.nodes["B"].first % map.nodes["B"].second %
+      map.nodes["C"].first % map.nodes["C"].second % map.nodes["D"].first % map.nodes["D"].second %
+      map.nodes["E"].first % map.nodes["E"].second;
+  std::string matrix = actor.matrix(req.str(), nullptr, &api);
+  rapidjson::Document d;
+  d.Parse(matrix.c_str());
+  rapidjson::Value a = d["sources_to_targets"].GetArray()[0].GetArray();
+
```

Comment:
This is a bit of weird place for testing matrix stuff. I'd suggest to glue it to the end of `test/matrix.cc` where the others live too. It's not gurka map enabled but there's other requests already.

Reply:
I'll try to move the test into `test/matrix.cc` today, thank you.

Reply:
Done, I've just added a new test into `test/matrix.cc`, and removed the changes from `gurka/test_time_tracking.cc`.